src/java.base/share/classes/java/util/zip/ZipFile.java

```diff
@@ -343,6 +343,9 @@ public ZipEntry getEntry(String name) {
      * Closing this ZIP file will, in turn, close all input streams that
      * have been returned by invocations of this method.
      *
+     * @implNote In the JDK implementation this method returns an
```

Comment:
The "In the JDK implementation" sounds a bit odd to me. I looked around some other `@implNote`s and many of of them use `@implNote This implementation ....`, so perhaps we should change this to:
``` 
@implNote This implementation returns an instance of {@link java.util.zip.InflaterInputStream}.
```

Reply:
Thanks for the suggestion. I've updated to code accordingly.